timed/reports/tests/test_notify_reviewers_unverified.py

```diff
@@ -27,14 +27,66 @@ def test_notify_reviewers(db, mailoutbox):
     ReportFactory.create(date=date(2017, 7, 1), task=task_no_work,
                          verified_by=reviewer_no_work)
 
-    call_command('notify_reviewers_unverified')
+    call_command(
+        'notify_reviewers_unverified',
+        '--cc=example@example.com',
+        '--message=This is a test'
+    )
 
     # checks
-    assert len(mailoutbox) == 1
     mail = mailoutbox[0]
+    assert len(mailoutbox) == 1
```

Comment:
I think this move happen by mistake right? assert len check should be before accessing mailoutbox otherwise we end up with a invalid index error.

Reply:
Yes this was a mistake. I have corrected it already.